docroot/themes/custom/mappy/mappy.theme

```diff
@@ -4,3 +4,13 @@
  * @file
  * Functions to support theming in the Mappy theme.
  */
+
+function mappy_page_attachments_alter(&$page) {
```

Comment:
I turned it off because it and phpmd kept having annoying arguments about whether or not I should be using camelCase. I guess I should turn phpcs back on...